CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Breaking changes:
 - Created Castle.Facilities.AspNet.SystemWeb facility so we can remove this from the Windsor core library. (@fir3pho3nixx, #283)
 
 New Features:
+- Created Castle.Facilities.AspNetCore facility to support Mvc web applications on dotnet core. (@fir3pho3nixx, #120)
```

Comment:
Did you get a chance to try this out using ASP.NET Core on .NET Framework?

Reply:
If not, feel free to just log an issue so we don't forget for v5 and defer it for now.

Reply:
I did check it for all of about 2-3 minutes before moving on to something else. Let me get back to you on this one. Let's hold out before raising the issue.

Reply:
@jonorossi - I looked at this again, it is definitely possible to run ASP.NET Core targeting [net461](https://github.com/fir3pho3nixx/Windsor.AspNetCore/commit/c21d456022294f5a128b5606872eced2f39f8485#diff-bc4f9cd406d7279d4b85cfeaff4a5403R1).  

The Visual Studio WebApp template appears to have a NuGet reference to [Microsoft.AspNetCore.All](https://github.com/fir3pho3nixx/Windsor.AspNetCore/commit/c21d456022294f5a128b5606872eced2f39f8485#diff-bc4f9cd406d7279d4b85cfeaff4a5403L8) which misleadingly targets netcoreapp2.0 instead of netstandard2.0. Once I added in the individual NuGets I needed it ran without any issues.

Reply:
> The Visual Studio WebApp template appears to have a NuGet reference to Microsoft.AspNetCore.All which misleadingly targets netcoreapp2.0 instead of netstandard2.0. Once I added in the individual NuGets I needed it ran without any issues.

The `Microsoft.AspNetCore.All` package [by design](https://github.com/aspnet/Announcements/issues/243) is .NET Core only, I think also one of the reasons they have introduced `Microsoft.AspNetCore.App`.